assets/airports/klas.json

```diff
@@ -6,7 +6,7 @@
     },
     "icao": "KLAS",
     "iata": "LAS",
-    "magnetic_north": 11.9,
+    "magnetic_north": 11.58,
```

Comment:
I would stick to 11.9, since that's what's listed on FAA charts. Even if it doesn't make any noticeable difference

Reply:
Okay, I thought this is kind of outdated therefore I used the NOAA calculator to get the updated value.

Reply:
It is outdated, but all the headings in the procedures are based on that number. Every few years (or whenever they consider the magnetic deviation to be too outdated) they updated it and all the procedures that are affected by jt